static/elements/chromedash-gantt.js

```diff
@@ -0,0 +1,206 @@
+        height: 36px;
+        text-align: center;
+        margin: auto;
+        border: 2px solid white;
+      }
+      .dev_trial {
+        background: #cfe2f3ff;
+      }
+      .origin_trial {
+        background: #6fa8dcff;
+      }
+      .shipping {
+        background: #0b5394ff;
+        color: white;
+      }
+      .stable {
+        border-color: #444;
+      }
+      .diamond span {
+        font-size: 14px;
+        font-weight: bold;
+        display: table-cell;
+        transform: rotate(-45deg);
+        width: 34px;
+        height: 34px;
+        vertical-align: middle;
+      }
+      .offset_0 { padding-left: 0px; }
+      .offset_1 { padding-left: 8px; }
+      .offset_2 { padding-left: 16px; }
+      .offset_3 { padding-left: 24px; }
+    `];
+  }
+
+  firstPhrase(milestone) {
+    if (milestone <= this.stableMilestone) {
+      return 'First milestone with this feature';
+    } else {
+      return 'First expected milestone with this feature';
+    }
+  }
+
+  renderDevTrial(milestone) {
+    if (!milestone) return nothing;
+    return html`
+      <div class="diamond dev_trial
+                  ${milestone == this.stableMilestone ? 'stable' : ''}"
+       title="${this.firstPhrase(milestone)} available to developers behind a flag"
+      ><span>${milestone}</span></div>`;
+  }
+
+  renderOriginTrial(milestone) {
+    if (!milestone) return nothing;
+    return html`
+      <div class="diamond origin_trial
+                  ${milestone == this.stableMilestone ? 'stable' : ''}"
+       title="${this.firstPhrase(milestone)} enabled on participating origins"
+      ><span>${milestone}</span></div>`;
+  }
+
+  renderShipping(milestone) {
+    if (!milestone) return nothing;
+    return html`
+      <div class="diamond shipping
+                  ${milestone == this.stableMilestone ? 'stable' : ''}"
+       title="${this.firstPhrase(milestone)} enabled by default"
+      ><span>${milestone}</span></div>`;
+  }
+
+  renderRow(
+    platform, devTrialMilestone, originTrialMilestone, shippingMilestone,
+    sortedMilestones) {
+    if (!devTrialMilestone && !originTrialMilestone && !shippingMilestone) {
+      return nothing;
+    }
+
+    let dtOffset = sortedMilestones.dt.indexOf(devTrialMilestone);
+    let otOffset = sortedMilestones.ot.indexOf(originTrialMilestone);
+    let shipOffset = sortedMilestones.ship.indexOf(shippingMilestone);
+
+    return html`
+       <ul class="bar ${originTrialMilestone ? 'with-ot' : 'without-ot'}">
+         <li class="platform">${platform}</li>
+         <li class="${'offset_' + dtOffset}">
+           ${this.renderDevTrial(devTrialMilestone)}
+         </li>
+         <li class="${'offset_' + otOffset}">
+           ${this.renderOriginTrial(originTrialMilestone)}
+         </li>
+         <li class="${'offset_' + shipOffset}">
+           ${this.renderShipping(shippingMilestone)}
+         </li>
+       </ul>
+    `;
+  }
+
+  render() {
+    let f = this.feature;
```

Comment:
All of these declarations are never reassigned. Maybe use `const` rather than `let`

Reply:
Good point.  Done.